deploy/submariner/lighthouse_operator_cluster_role.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
We need to move this file to deploy/ (it's nos submariner specific), it's not lighthouse specific, it's operator specific.

I suggest that we name it "deploy/cluster_role.yaml"

Reply:
Done.